pkg/operator/operator.go

```diff
@@ -92,10 +92,17 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 
 	// Start controllers
 	klog.Info("Starting controller-manager")
-	go o.manager.Start(ctx.Done())
+	go func() {
+		err := o.manager.Start(ctx.Done())
+		if err != nil {
+			klog.Fatalf("Failed to start controller-runtime manager: %v", err)
```

Comment:
Would it make sense to close the ctx.Done() channel so the operator reboots (assuming this is the entry point) ?

Reply:
I tried to do this, but ctx.Done() is not closable (it's a <-chan).

Reply:
Ah, then I guess it's fine as it is.
An alternative would be to create a context.WithCancel starting from ctx, not sure it's worth the effort